client/my-sites/plans-features-main/index.tsx

```diff
@@ -784,6 +787,7 @@ const PlansFeaturesMain = ( {
 								layoutClassName="plans-features-main__plan-type-selector-layout"
 								enableStickyBehavior={ enablePlanTypeSelectorStickyBehavior }
 								stickyPlanTypeSelectorOffset={ masterbarHeight }
+								coupon={ coupon }
```

Comment:
Actually, sorry - this doesn't look like it's needed as it's added to `planTypeSelectorProps` above